app.py

```diff
@@ -145,4 +145,4 @@ def index():
 
 
 if __name__ == '__main__':
-    app.run(debug=True)
+    app.run(debug=True, port=9191)
```

Comment:
- Because there are two places need to edit (i.e. `app.py` and `readme.md`).  
- I don't familiar with uwsgi file, so I just keep it like it was.   

Why don't you make a pull request, so I can close mine?